samples/kotlin-android-app-destinations/src/main/java/com/segment/analytics/destinations/plugins/FirebaseDestination.kt

```diff
@@ -103,12 +103,14 @@ class FirebaseDestination(
             eventName = makeKey(eventName)
         }
 
+        var bundledProperties: Bundle?
         payload.properties.let {
-            val formattedProperties = formatProperties(it)
-            firebaseAnalytics.logEvent(eventName, formattedProperties)
-            analytics.log("firebaseAnalytics.logEvent($eventName, $formattedProperties)")
+            bundledProperties = formatProperties(it)
         }
```

Comment:
more idiomatic to write
```
val bundledProperties = payload.properties?.let {
  formatProperties(it)
}
```

Reply:
Cool but it'd be ...
`bundledProperties = payload.properties?.let {
  formatProperties(it)
}`